docs/05-internals/design/adr/ADR-0006-use-zio-http.md

```diff
@@ -0,0 +1,20 @@
+# ADR-0006 Use ZIO HTTP
+
+Date: 2022-12-01
+
+## Status
+
+Accepted
+
+## Context
+
+The current routes use the `Akka Http` framework. Because of Lightbend's decision that `Akka` will no longer be open source and their introduction of a commercial license, we want to get away from `Akka Http`. This also fits the general strategic decision to use ZIO for the backend.
```

Comment:
```suggestion
The current routes use the `Akka Http` library. Because of changes to the licensing of the `Akka` framework, we want to move away from using `Akka Http`. This also fits the general strategic decision to use ZIO for the backend.
```